plugins/org.preesm.codegen/src/org/preesm/codegen/xtend/task/CodegenEngine.java

```diff
@@ -380,4 +386,17 @@ private void format(final IFile iFile) {
     }
 
   }
+
+  /**
+   * 
+   * @param apolloFlag
+   *          Enable the use of Apollo for intra-actor optimization
+   */
+  public void registerApollo(String apolloFlag) {
+    if (apolloFlag != null && apolloFlag.equalsIgnoreCase("true")) {
```

Comment:
rewrite: this.apolloEnabled = (apolloFlag != null && "true".equalsIgnoreCase(apolloFlag))

Reply:
Changed